protocols/CopyToResultsDir.sh

```diff
@@ -29,16 +29,16 @@ umask 0007
 
 #Function to check if array contains value
 array_contains () {
```

Comment:
array_contains function is present in multiple protocols: should we move this to the header for less redundant code? In that case it will also get included in protocols that do not need it, which may be confusing...